chain/chain/src/types.rs

```diff
@@ -171,7 +171,9 @@ impl BlockEconomicsConfig {
     const MAX_GAS_MULTIPLIER: u128 = 20;
```

Comment:
as a part of this NEP -- we'll probably also have to increase this multiplier (the discussion is still pending).
But if we do that, we should do it in the same PR.